renderdoc/api/replay/pipestate.h

```diff
@@ -383,10 +383,10 @@ For some APIs that don't distinguish by entry point, this may be empty.
 
   DOCUMENT(R"(Retrieves the read/write resources bound to the depth-stencil output.
 
-:return: The currently bound depth-stencil resource.
-:rtype: BoundResource
+:return: The currently bound depth-stencil resources.
+:rtype: List[BoundResource]
 )");
-  BoundResource GetDepthTarget() const;
+  rdcarray<BoundResource> GetDepthTargets() const;
```

Comment:
I don't want to change this interface like this. For 99% of cases there will only be one depth target and making everyone using this API handle that when they probably don't care about the depth resolve target is an unnecessary burden. Almost everyone is going to have to undo this call to just get the depth target they care about - e.g. this will break any of the autotests that want to query the depth target as well as the 'Where is my Draw' extension. There's an argument that the output targets shouldn't include resolve targets either, but at least in that case it's already a list of multiple outputs.

In this case I think it's better to have a (slightly) leaky abstraction and add a separate `GetDepthResolveTarget` instead which can be used when needed.

Reply:
Makes sense, earlier I wasn't sure about changing the existing function or making a new one. I'll update the PR today.